pkg/controllers/dynakube/extension/reconciler_test.go

```diff
@@ -0,0 +1,92 @@
+package extension
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/scheme/fake"
+	dynatracev1beta3 "github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```

Comment:
```suggestion
	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
```
Please read new section I've added recently https://github.com/Dynatrace/dynatrace-operator/blob/main/doc/coding-style-guide.md#naming

Reply:
[Fixed here](https://github.com/Dynatrace/dynatrace-operator/pull/3485/commits/f288a82fe537eb2e859f3dca4fd369fafcf96094)